jekyll/_cci2/deployment-examples.md

```diff
@@ -0,0 +1,653 @@
+orbs:
+  aws-code-deploy: circleci/aws-code-deploy@x.y.z # Use the AWS CodeDeploy orb in your config
+
+workflows:
+  deploy_application:
+    jobs:
+      - aws-code-deploy/deploy:
+          application-name: myApplication # The name of an AWS CodeDeploy application associated with the applicable IAM user or AWS account.
+          deployment-group: myDeploymentGroup # The name of a new deployment group for the specified application.
+          service-role-arn: myDeploymentGroupRoleARN # The service role for a deployment group.
+          bundle-bucket: myApplicationS3Bucket # The s3 bucket where an application revision will be stored.
+          bundle-key: myS3BucketKey # A key under the s3 bucket where an application revision will be stored.
+```
+
+## Azure Container Registry
+
+This section describes a simple deployment to the Azure container registry (ACR) using the CircleCI ACR orb and `version 2.1` configuration.
+
+For detailed information about the Azure ACR orb, including all options, refer to the [CircleCI ACR Orb Reference](https://circleci.com/orbs/registry/orb/circleci/azure-acr) page.
+
+1. Whether your require a user or service principal login, you will need to provide environment variables for username, password and tennent to CircleCI. For user logins use env var names as follows: `AZURE_USERNAME`, `AZURE_PASSWORD` and `AZURE_TENANT`. For service principal logins use: `AZURE_SP`, `AZURE_SP_PASSWORD` and `AZURE_SP_TENANT`. {% include snippets/env-var-or-context.md %}
+
+2. Use the orb's `build-and-push-image` job to build your image and deploy it to ACR. Note the use of [workflows]({{ site.baseurl }}/2.0/workflows/) to deploy only if the current branch is `master`.
+  
+```
+version: 2.1 # Use version 2.1 config to get access to orbs, pipelines
+
+orbs:
+  azure-acr: circleci/azure-acr@x.y.z # Use the Azure ACR orb in your config
+
+workflows: 
+  build-deploy:
+    jobs:
+      - azure/build-and-push-image:
+          dockerfile: <name-of-your-dockerfile> # defaults to `Dockerfile`
+          path: <path-to-your-dockerfile> # Defaults to working directory
+          login-server-name: <your-login-server-name> # e.g. {yourregistryname}.azure.io
+          registry-name: <your-ACR-registry-name>
+          repo: <URI-to-your-login-server-name>
+          filters:
+            branches:
+              only: master # Only deploys when the commit is on the Master branch
+```
+
+If pushing to your repo is required, see the [Adding Read/Write Deployment Keys to GitHub or Bitbucket]( {{ site.baseurl }}/2.0/gh-bb-integration/) section of the GitHub and Bitbucket Integration document for instructions. Then, configure the Azure Web App to use your production branch.
+
+## Capistrano
+
+Integrating CircleCI with Capistrano is simple. Once your project is set up to use Capistrano, just run [deployment commands](https://github.com/capistrano/capistrano/blob/master/README.md#command-line-usage) within your CircleCI job steps as required.
```

Comment:
Opinionated comment: I would maybe remove the "is simple" line just to be inclusive of user's of all experience levels.

Reply:
Oh yeah good point!